ddtrace/contrib/logbook/patch.py

```diff
@@ -0,0 +1,74 @@
+import logbook
+
+import ddtrace
+from ddtrace import config
+
+from ...internal.utils import get_argument_value
+from ...vendor.wrapt import wrap_function_wrapper as _w
+from ..logging.constants import RECORD_ATTR_ENV
+from ..logging.constants import RECORD_ATTR_SERVICE
+from ..logging.constants import RECORD_ATTR_SPAN_ID
+from ..logging.constants import RECORD_ATTR_TRACE_ID
+from ..logging.constants import RECORD_ATTR_VALUE_EMPTY
+from ..logging.constants import RECORD_ATTR_VALUE_ZERO
+from ..logging.constants import RECORD_ATTR_VERSION
+from ..trace_utils import unwrap as _u
+
+
+config._add(
+    "logbook",
+    dict(),
+)
+
+
+def get_version():
+    # type: () -> str
+    return getattr(logbook, "__version__", "")
+
+
+def _tracer_injection(event_dict):
```

Comment:
You can do this as a follow up: We can refactor this code. It is duplicated in our other logging integrations

Reply:
Noted